### ISSUE TITLE
Setting Keyboard Shortcut for paste-node-as-clone

### ISSUE THREAD

**Issue by Ally-B** [leo-editor/leo-editor#3896]
Leo 6.7.8 on Windows 10

I've added this shortcut in myLeoSettings.leo @settings/@keys/@shortcuts

-------------------------------------------------------------------------
# You can define keyboard shortcuts here of the form:
#
#    some-command Shift-F5

paste-node-as-clone = Shift-F8
---------------------------------------------------------------------------

But it doesn't work. When I open a command minibuffer it doesn't recognise paste-node-as-clone as a command. 
I feel I must be doing something wrong!?

ta

**Comment by LewisNeal** [comment]
Is it possible the _paste-node-as-clone_ command is obsolete?
I could not find it in LeoPyRef.leo or LeoDocs.leo
However the _clone-node_ command is found in both.

Using the F11 Help for command:
 F11 paste-node-as-clone
 no docstring available.

 F11 clone-node
 Create a clone of the selected outline.
 
Using the Menu does nothing:
 Outline>Paste Node As Clone

**Comment by tbpassin** [comment]
My `@shortcuts` tree is not under `@keys`.  It's directly under `@settings`. Try raising yours.  Also, try restarting Leo rather than just reloading the settings.

**Comment by Ally-B** [comment]
Hi Gents, 
Thanks for such speedy replies! 
You could be right Lewis but if I use Ctrl+Shift+C to copy the node then the menu item Outline>Paste Node As Clone works as you would expect. And it is kind of handy because I can create the clone at the point I want to, rather than create it then move it. 
It is interesting that Paste Node As Clone doesn't have hyphens... does that mean it never was a command available in the mini-buffer?

Hi Thomas -
I tried moving it directly under @settings and restarting Leo but no joy. It obviously isn't a deal breaker just a nice to have, and there can't be hundreds of people wanting it!

Thanks again. 

Ally

**Comment by tbpassin** [comment]
The actual command name seems to be _paste-retaining-clones_.  Give that a try, see if it works for you.

Here's how I found it. I opened LeoSettings.leo and searched for the name on the menu item: _Paste Node As Clone_. That gave me a node with context menu data that included the line `paste-retaining-clones Paste Node As Clone`.  

Then I used tab-completion in the minibuffer for _paste_ and got these matching commands:
```
                paste-as-template
 Ctrl+Shift+v   paste-node
                paste-retaining-clones
 Ctrl+v         paste-text
```
The same command comes up here, so it looks like it's the right one.
